### DISCUSSION TITLE
PAG用BMP预合成尺寸问题

### DISCUSSION THREAD

**Discussion by bennettyhk** [Tencent/libpag#1648] [Q&A] [answered]
我AE里面画布尺寸是60帧1384的宽度，然后用PAG插件导出之后同样的尺寸不同文件。。。他们导出的大小都不一样，而且帧率都变成24帧了，请帮忙解决一下。。。急急急~~
AE的文件大小是这样的
![image](https://github.com/Tencent/libpag/assets/44016566/5ad50bf8-877e-4f00-8b1a-46ce47e02b08)

PAG的文件大小是这样的
![image](https://github.com/Tencent/libpag/assets/44016566/8ed77917-b638-4d1e-9bce-1a4b79d768f4)

用lottie（含图片）模式是可以的尺寸也没有问题，用PAG就不行。。。而且在导出的时候也是识别出60帧的
![image](https://github.com/Tencent/libpag/assets/44016566/f28ef343-d577-4e80-a9e7-c5f3b2544ab0)

**Comment by kevingpqi123** [comment, marked as answer]
可以看下这篇文档：https://pag.art/docs/plugin-config.html 中的 “BMP 预合成配置”

BMP预合成相当于一个视频，视频解码属于耗资源的操作，不是帧率越高、尺寸越大越好，够用就好，否则终端性能压力较大

**Comment by bennettyhk** [comment]
开发需要的就是这个尺寸和帧率。。。有什么办法解决哇

**Reply by kevingpqi123** [reply]
你咋不看文档呢，文档上面的配置不可以修改吗
